youtube_dl/extractor/nick.py

```diff
@@ -84,17 +84,17 @@ class NickBrIE(MTVServicesInfoExtractor):
     _VALID_URL = r'''(?x)
                     https?://
                         (?:
-                            (?P<domain>(?:www\.)?nickjr|mundonick\.uol)\.com\.br|
+                            (?P<domain>(?:www\.)?nickjr|(?:www\.)?mundonick)\.com\.br|
```

Comment:
Will break on `mundonick` lookup in dict below.

Reply:
I keep getting ```ValueError: Input strings must be a multiple of 16 in length``` when testing NickBr. Is that related?